### ISSUE TITLE
Crossplatform tooltip system

### ISSUE THREAD

**Issue by objorke** [oxyplot/oxyplot#1418]
Make a cross platform solution that shows tooltips on
- titles
- annotations
- axes
- series

**Comment by objorke** [comment]
See discussion in #1081 and #1350 and PR #1415

**Comment by VisualMelon** [comment]
Silviubogan mentioned that Legends could do with tool-tips as well.

**Comment by silviubogan** [comment]
I think that in my fork all the listed tasks are done excepting:

- [ ] separate tooltip for subtitle (related to #1420)
- [ ] legend tooltip/s
    - **A question:** Each legend sub-element should have its own tooltip or there should be only a single tooltip for all the legend area?
- [ ] support for other platforms (only WinForms and WPF are currently supported)

If there are any more tasks to add to this list, please let me know.

Thank you.

**Comment by objorke** [comment]
Yes, I think we should allow a separate tooltip for subtitles, but then we also need to add a `PlotModel.SubtitleTooltip`. We currently only have https://github.com/oxyplot/oxyplot/blob/develop/Source/OxyPlot/PlotModel/PlotModel.cs#L683

Could the legend sub-elements simply reuse the tooltips from the series? Otherwise I think we need to add a `LegendTooltip` on the series...

In this repository we only need to support WPF and WindowsForms, but I think it would be good to check if the changes here makes it possible to implement on the other platforms (first of all Xamarin Forms, I think).

**Comment by VisualMelon** [comment]
`LegendTooltip` is also requested in #1268
